src/books/Tests/Books.BusinessTests/Handlers/BookInstancesCommandsHandlersTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Xunit;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Books.Infrastructure.Contracts;
+using Books.Infrastructure.Models;
+using Books.Business.Commands;
+using Books.Business.Handlers;
+
+namespace Books.BusinessTests.Handlers
+{
+    public class BookInstancesCommandsHandlersTests
+    {
+        private readonly Mock<IBookRepository<BookRead>> _bookReadRepositoryMock;
+        private readonly Mock<IBookInstancesRepository<BookInstance>> _bookInstanceRepositoryMock;
+
+        public BookInstancesCommandsHandlersTests()
+        {
+            _bookReadRepositoryMock = new Mock<IBookRepository<BookRead>>();
+            _bookInstanceRepositoryMock = new Mock<IBookInstancesRepository<BookInstance>>();
+        }
+
+        [Fact]
+        public async void CreateBookInstanceHandlerTest()
+        {
+            // Arrange
+            CreateBookInstanceCommand request = new CreateBookInstanceCommand("11111111111", true);
+            _bookInstanceRepositoryMock.Setup(x => x.Create(It.IsAny<BookInstance>()));
+            CreateBookInstanceHandler handler = new CreateBookInstanceHandler(_bookInstanceRepositoryMock.Object);
+
+            // Act
+            await handler.Handle(request, default);
+
+            // Assert
+            _bookInstanceRepositoryMock.VerifyAll();
```

Comment:
Do not use `VerifyAll`. Specify what is verified and how many times. Fix for all other occurrences as well.